### PR TITLE
Spawn in the center

### PR DESCRIPTION
**Pull request by Serg-Bartosh**

### CODE REVIEW

File: src/figure_factory.py [hunk 1]
```diff
@@ -10,14 +10,14 @@ class FigureFactory:
     __kind_randomizer = FigureKindRandomizer()
 
     @staticmethod
-    def create(kind: FigureKind) -> Figure:
+    def create(kind: FigureKind, weight: int) -> Figure:
```
Comment (alena-bartosh) [inline src/figure_factory.py]: Lets rename like "`board_weight`" in all place in `FigureFactory`

Comment (s373r) [reply src/figure_factory.py]: in `create_random()` too

File: src/figure_factory.py [hunk 2]
```diff
@@ -10,14 +10,14 @@ class FigureFactory:
...
         start_rotation = RotationKind.ROTATION_0
         figure_points = FigureMover.get_points_after_rotation(kind, start_rotation)
 
-        return Figure(kind, Point(0, 0), figure_points, start_rotation)
+        return Figure(kind, Point(0, weight // 2 - 2), figure_points, start_rotation)
```
Comment (s373r) [inline src/figure_factory.py]: extract `weight // 2 - 1` (correct to `- 1`) to variable `center_position`

**Review by s373r — CHANGES_REQUESTED** [review]

**Review by alena-bartosh — DISMISSED** [review]
Good job

File: src/figure_factory.py (hunk 2 above)
Comment (s373r) [reply src/figure_factory.py]: not done

**Review by s373r — APPROVED** [review]

File: src/figure_factory.py (hunk 1 above)
Comment (Serg-Bartosh) [reply src/figure_factory.py]: done

File: src/figure_factory.py (hunk 2 above)
Comment (Serg-Bartosh) [reply src/figure_factory.py]: done

**Review by alena-bartosh — APPROVED** [review]

**Pull request merged**